CHANGES_NEXT_RELEASE

```diff
@@ -1 +1,3 @@
+ADD: upgrade docker based image from Tomcat8 to Tomcat9
+ADD: migrate log4j v1 to v2 (#184)
```

Comment:
```suggestion
ADD: migrate log4j v1 (1.7.25) to v2 (2.17.1) (#184)
```

Should we mention the upgrade in junit in a similar way? (In nodejs-based projects we include devDepedencies in the upgrade and this would be a similar case...)